app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -98,13 +99,13 @@ export default {
       const sortDatasets = this.items
       const allDatasets = sortDatasets.sort((a, b) => a.attributes.name.localeCompare(b.attributes.name));
       let slug = this.$route.params.id
-      const indexToggle = allDatasets.findIndex(dataset => dataset.attributes.slug == slug)
+      const indexToggle = allDatasets.findIndex(dataset => dataset.attributes.slug === slug)
       this.toggle = indexToggle
       if (this.toggle === -1) {
         this.toggle = 0
         slug = slug = allDatasets.length ? allDatasets[0].attributes.slug : ''
       }
-      let firstElement = allDatasets.find(dataset => dataset.attributes.slug == slug)
+      let firstElement = allDatasets.find(dataset => dataset.attributes.slug === slug)
       let filteredArray = allDatasets.filter(dataset => dataset.attributes.slug !== slug)
```

Comment:
Apply destructuring all over this file. Example:

```
let { id } = this.$route.params || {}
const indexToggle = allDatasets.findIndex(({ attributes: { slug } = {}}) => slug === id)
```

You can only trust on those objects from external libraries, i.e. vue-router exposes always a `params` property.

Reply:
Do this kind: `allDatasets[0].attributes.slug` should be forbidden by law

Reply:
@jorgeatgu this is the only comment left to merge the PR